### PR TITLE
Removed space before !

### PR DESCRIPTION
**Pull request by dave9123**

### CODE REVIEW

**Review by dave9123 — COMMENTED** [review]
Removed space before ! in most languages

File: common/resources/assets/negativity/Negativity_fr_FR.properties [hunk 1]
```diff
@@ -2,16 +2,16 @@ prefix=&a[&2Negativity&a]
 
 report_wait=Merci d'attendre une seconde entre 2 reports
 
-invalid_player=%arg% n'est pas un joueur !
-not_forget_player=N'oubliez pas le joueur !
+invalid_player=%arg% n'est pas un joueurs!
```
Comment (RedNesto) [inline common/resources/assets/negativity/Negativity_fr_FR.properties:5]: French actually has spaces before exclamation marks (and question marks) so please revert these changes in both fr_FR files.
The added s is also incorrect btw. It was fine before.

**Review by RedNesto — CHANGES_REQUESTED** [review]

File: common/resources/assets/negativity/Negativity_fr_FR.properties (hunk 1 above)
Comment (dave9123) [reply common/resources/assets/negativity/Negativity_fr_FR.properties:5]: Oh ok
